src/Microsoft.PowerShell.Commands.Management/commands/management/Hotfix.cs

```diff
@@ -66,52 +66,56 @@ public sealed class GetHotFixCommand : PSCmdlet, IDisposable
         private ManagementObjectSearcher _searchProcess;
 
         private bool _inputContainsWildcard = false;
+        private readonly ConnectionOptions _connectionOptions  = new ConnectionOptions
+            {
+                Authentication = AuthenticationLevel.Packet,
+                Impersonation = ImpersonationLevel.Impersonate,
+                Username = Credential?.UserName,
+                SecurePassword = Credential?.Password
```

Comment:
I think we could move the initialization to BeginProcessing() (or ProcessRecord() if we want separate credential for every computer).

Reply:
But how will we specify separate creds for each computer, it doesn't take input via pipeline.

Reply:
If we don't want the scenario we can move to BeginProcessing().

Reply:
so a I'll

```
 private readonly ConnectionOptions _connectionOptions  = new ConnectionOptions {};
```

then in Begin
```
            conOptions.Authentication = AuthenticationLevel.Packet;	
            conOptions.Impersonation = ImpersonationLevel.Impersonate;	
            conOptions.Username = Credential?.UserName;	
            conOptions.SecurePassword = Credential?.Password;
```

(not with my laptop .) Is the above correct ?

Reply:
I think yes.